libs/domain/picking/src/components/decorators-test/stories/test.stories.ts

```diff
@@ -0,0 +1,42 @@
+import {
+  PickingListEntity,
+  PickingListOffline,
+} from '@spryker-oryx/picking/offline';
+import { Meta, Story } from '@storybook/web-components';
+import { html, TemplateResult } from 'lit';
+import { storybookPrefix } from '../../../../.constants';
+
+export default {
+  title: `${storybookPrefix}/Decorators test`,
+  parameters: { chromatic: { disableSnapshot: true } },
+} as Meta;
+
+const Template: Story = (): TemplateResult => {
+  const testOfflineData = {
+    items: [
+      {
+        productId: 'test',
+        product: {
+          id: 'test',
+          sku: 'test',
+          productName: 'test',
+          image: null,
+          imageLarge: null,
+        },
+      },
+    ],
+    itemsCount: 1,
+    orderReferences: ['test'],
+    productSkus: ['test'],
+    requestedDeliveryDate: new Date(),
+    localStatus: 'test',
+    id: 'test',
+    status: 'test',
+    createdAt: new Date(),
+    updatedAt: new Date(),
+  };
+  const entity = new PickingListEntity(testOfflineData as PickingListOffline);
```

Comment:
I think it's not enough to just create an entity instance.
We need to validate that the entity metadata is properly collected and for that we should try to store it in the IndexedDb and retrieve it back from it.
You can use `IndexedDbService` for this.